### ISSUE TITLE
ca-management - undefined method Error for YCP::YaPI

### ISSUE THREAD

**Issue by vmoravec** [yast/ycp-killer#465]
Yast module `ca-management` crashes on various places in different situations, like when I provide invalid CA password  I get 

```
ycp/wfm.rb:85 Client call failed with undefined method Error' for YCP::YaPI:Module and backtrace
/usr/share/YaST2/include/ca-management/popup.rb:75:in showErrorCaManagement', 
/usr/share/YaST2/include/ca-management/util.rb:295:in getPassword', 
/usr/share/YaST2/include/ca-management/startup.rb:222:in Startup', 
/usr/share/YaST2/include/ca-management/wizards.rb:89:in block in MainSequence', 
/usr/lib64/ruby/vendor_ruby/2.0.0/yast/builtins.rb:514:in call', 
/usr/lib64/ruby/vendor_ruby/2.0.0/yast/builtins.rb:514:in eval', 
/usr/share/YaST2/modules/Sequencer.rb:279:in WS_run', 
/usr/share/YaST2/modules/Sequencer.rb:344:in Run', 
/usr/share/YaST2/include/wizard/sequencer.rb:45:in WizardSequencer', 
/usr/share/YaST2/include/ca-management/wizards.rb:121:in MainSequence', 
/usr/share/YaST2/clients/ca_mgm.rb:197:in main', 
/usr/share/YaST2/clients/ca_mgm.rb:258:in <top (required)>', 
/usr/lib64/ruby/vendor_ruby/2.0.0/yast/wfm.rb:82:in eval'", 
/usr/lib64/ruby/vendor_ruby/2.0.0/yast/wfm.rb:82:in run_client'
```

**Comment by jsuchome** [comment]
That message you mention at the end is known product of bug 799660. (Not related to ruby translation)

**Comment by lslezak** [comment]
BTW `limal` has been dropped in Factory...

**Comment by vmoravec** [comment]
The perl issue seems to be resolved, I edited the issue description to match the recent status.

**Comment by jreidinger** [comment]
ca-management somehow attach via perl bindings method directly to YaPI namespace, which is problem. I try to discuss it with @mvidner as perl-bindings in involved in it ( ycp doesn't allow it )

**Comment by jreidinger** [comment]
Fixed in ruby-bindings 1.1.4. Please reopen if it still occurs.
